src/components/MapContent.vue

```diff
@@ -1,212 +1,20 @@
 <template>
     <div style="height: 100%;width:100%">
-      <el-container style="height:100%;background:white;">
-        <el-header style="text-align: left; font-size: 14px;padding:0" height="40px" class="dialog-header">
-          <DialogToolbarContent :activeId="activeDockedId" :dialogTitles="dockedArray"
-            :showIcons="entries[findIndexOfId(activeDockedId)].mode!=='main'" @maximise="dockedMaximise"
-            @minimise="dockedMinimise" @close="dockedClose" @titleClicked="dockedTitleClicked"/>         
-        </el-header>
-        <el-main class="dialog-main">
-          <div style="width:100%;height:100%;position:relative;overflow:hidden;">
-            <FloatingDialog v-for="item in entries" :entry="item" :index="item.id"
-              :key="item.id" v-on:mousedown.native="dialogClicked(item.id)"
-              @maximise="dialogMaximise(item.id)" @minimise="dialogMinimise(item.id)" 
-              @close="dialogClose(item.id)"/>
-          </div>
-        </el-main>
-      </el-container>
+      <FloatingFlow />
```

Comment:
Does MapContent.vue do anything now that the Dialogs are removed?

Reply:
It will be used to contain different workflow/prototype.

Reply:
The idea is that MapContent should contain the information of the array of viewers which will then get passed into different workflows. (Allowing toggling of different workflows.)

Reply:
We are not there yet but we will be.

Reply:
Yep I see so you have a premade workflow and pas that to MapContent